Add business support CTA to admin panel

bitbager opened the pull request:
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.13 <!-- see the comment below -->                  |
| Bug fix?        | no                                                      |
| New feature?    | no                                                      |
| BC breaks?      | no                                                      |
| Deprecations?   | no<!-- don't forget to update the UPGRADE-*.md file --> |
| License         | MIT                                                          |

<!--
 - Bug fixes must be submitted against the 1.12 branch
 - Features and deprecations must be submitted against the 1.13 branch
 - Make sure that the correct base branch is set

 To be sure you are not breaking any Backward Compatibilities, check the documentation:
 https://docs.sylius.com/en/latest/book/organization/backward-compatibility-promise.html
-->

Jibbarth committed:
[Checkout] Prevent extra fields when login in on addressing page

bitbager committed:
Add business support CTA to admin panel

diimpp commented:
![image](https://github.com/Sylius/Sylius/assets/870747/c7ba44b7-b23b-4a06-8676-0e755a1e857c)
At least it needs indication icon of `external url`
![image](https://github.com/Sylius/Sylius/assets/870747/87fb98bd-df1e-482a-bd02-07a7e98e6b8d)

What about this section? Looks like new URLs should be placed over here firstly. 
![image](https://github.com/Sylius/Sylius/assets/870747/8f3d38b5-0f7a-4e51-859e-15275ca56498)

Jibbarth commented on src/Sylius/Bundle/AdminBundle/Resources/translations/messages.fr.yml:
```diff
@@ -47,6 +47,11 @@ sylius:
                     header: Ventes
                     orders: Commandes
+                support:
+                    header: Soutien
```
`Support` can/should be used in this context

loic425 replied:
I agree, Support should be used in this case.

bitbager committed:
Add all translations & change label

bitbager commented:
> ![image](https://user-images.githubusercontent.com/870747/275348906-c7ba44b7-b23b-4a06-8676-0e755a1e857c.png) At least it needs indication icon of `external url` ![image](https://user-images.githubusercontent.com/870747/275348973-87fb98bd-df1e-482a-bd02-07a7e98e6b8d.png)
> 
> What about this section? Looks like new URLs should be placed over here firstly. ![image](https://user-images.githubusercontent.com/870747/275348883-8f3d38b5-0f7a-4e51-859e-15275ca56498.png)

Valid points. I changed the menu item group completely and added the target="_blank" to each link

jakubtobiasz commented:
/bns:deploy

jakubtobiasz commented:
/bns:deploy

jakubtobiasz commented:
/bns:deploy

damonsson commented:
/bns:deploy

damonsson commented:
/bns:deploy

Jibbarth committed:
Update src/Sylius/Bundle/CoreBundle/Form/Type/Checkout/AddressType.php

Co-authored-by: Victor Vasiloi <victor.vasiloi@gmail.com>

Jibbarth committed:
Add tests that form doesn't display Extra fields error after login in checkout

Jibbarth committed:
Remove fr translation

jakubtobiasz committed:
bugfix #15435 [Checkout] Prevent extra fields when login in on addressing page (Jibbarth)

This PR was merged into the 1.12 branch.

Discussion
----------

| Q               | A   |
|-----------------|-----|
| Branch?         | 1.12 |
| Bug fix?        | yes |
| New feature?    | no |
| BC breaks?      | no |
| Deprecations?   | no |
| Related tickets | fixes #15183 |
| License         | MIT |

Refs video in #15183

When we login in addressing page after submit once the page, the reload process after login resubmit data, and keep sending `form[customer][email]`, that it doesn't be added in form. 

So I unset these data on pre-submit. 
I also change the csrf_error message for something more "understandable".

By this way, we can still have previous submitted data for address fields.

![image](https://github.com/Sylius/Sylius/assets/3168281/e53f0c5a-8fe7-4bd7-81b9-021eb8d819b7)

The other way, suggested by @Wojdylak, is to change this line https://github.com/Sylius/Sylius/blob/1.12/src/Sylius/Bundle/UiBundle/Resources/private/js/sylius-api-login.js#L50 
by 
```
window.location = window.location.href;
``` 

But using it, all already filled data are lost.

WDYT ? Would it be acceptable to lose such data, or should we keep them ? 

Commits
-------
  [Checkout] Prevent extra fields when login in on addressing page
  Update src/Sylius/Bundle/CoreBundle/Form/Type/Checkout/AddressType.php
  Add tests that form doesn't display Extra fields error after login in checkout
  Remove fr translation

GSadee commented:
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "master"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

GSadee commented:
Closing in favour of #15642
